time ts
float64 fovy
float64 aspect
float64 camera_rate
float64 face_detect_rate
float64 hand_detect_rate
float64 saliency_detect_rate
